cpu/cc26x0/periph/rfc.c

```diff
@@ -131,6 +131,7 @@ bool rfc_ping_test(void)
     pingCommand.commandID = CMDR_CMDID_PING;
     RFC_DBELL->CMDR = (uint32_t) (&pingCommand);
     while (!RFC_DBELL->CMDSTA); /* wait for cmd execution */
+    printf("rfc_ping fails without a printf reading cmdsta %lu\n", RFC_DBELL->CMDSTA);
```

Comment:
Not sure if it's related, but I observed print statements in isr
functions  (at leat I interpret functions starting with isr_ as such).
AFAIK printing in interrupt routines is never a good idea. If you need some
sort of signaling and the interrupts occur only sparsely then you can
toggle a led or use another variable to somehow mark that an interrupt was
thrown and do the printing in normal execution
Am 03.06.2016 10:39 schrieb "Leon George" <notifications@github.com>:

> In cpu/cc26x0/periph/rfc.c
> <https://github.com/RIOT-OS/RIOT/pull/5496#discussion_r65674010>:
>
> > @@ -131,6 +131,7 @@ bool rfc_ping_test(void)
> >      pingCommand.commandID = CMDR_CMDID_PING;
> >      RFC_DBELL->CMDR = (uint32_t) (&pingCommand);
> >      while (!RFC_DBELL->CMDSTA); /* wait for cmd execution */
> > +    printf("rfc_ping fails without a printf reading cmdsta %lu\n", RFC_DBELL->CMDSTA);
>
> note that the mere presence of this line affects the evaluation in the
> interrupt which happens before the printf is reached oO
> can't wait to see if anyone finds the reason
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/RIOT-OS/RIOT/pull/5496/files/32f992e5f89b3701bfa6c3de99e0ed2bd6b8e0f6..7bf8ed592f43a05a2d87ef83286c3950b8db726a#r65674010>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAtHOREcFUflfXwoyi72dPtfYc7XbK9Fks5qH-hXgaJpZM4Isg30>
> .
>